index.html

```diff
@@ -13,6 +13,7 @@
     <div class="container">
       <div class="container-img-book">
         <img class="book-img" src="./assets/book-cover.png" alt="Book cover" />
+        <img class="book-badge" src="./assets/badge.png" alt="badge" />
```

Comment:
Try to add it using only css with an ::before or ::after selector. Remember html have to be descriptive of the layout of the component, decorations should be done using css.